index.html

```diff
@@ -4,10 +4,37 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">   
     <link rel="stylesheet" href="style.css">
     <title>Portfolio</title>
 </head>
 <body>
-    <div class="header"></div>
+    <div class="header">
+        <img src="/image/images.png" width="100px">
+        <a href="#" class="fa fa-bars"></a>
+    </div>
+
+    <div class="menu">
+        <ul class="menu">
+            <li><a href="#home" class="menu-btn">Home</a></li>
+            <li><a href="#about" class="menu-btn">About</a></li>
+            <li><a href="#services" class="menu-btn">Services</a></li>
+            <li><a href="#skills" class="menu-btn">Skills</a></li>
+            <li><a href="#contact" class="menu-btn">Contact</a></li>
+        </ul>
+    </div>
+
+    <div class="hero">
+        <h1>I'm Abhishek. <br> Glad to see <br> you !</h1>
+        <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. </p>
+        <a href="https://www.linkedin.com/in/abhishek-kausik-600aa6218/">LET'S CONNECT</a>
+            <ul class="social">
+                <li><a href="https://m.facebook.com/abhisekh.kaushik.9?eav=AfaA4ohBeaoQ7IVUNJE5Nbz2VYSv_cPf7PJDGNyp-G9M9dzmuaBOBWLsNX-WrA9ha-Y&paipv=0" class="fa fa-facebook"></a></li>
+                <li><a href="https://www.instagram.com/sketcher_abhii/" class="fa fa-instagram"></a></li>
+                <li><a href="https://www.linkedin.com/in/abhishek-kausik-600aa6218/" class="fa fa-linkedin"></a></li>
+                <li><a href="https://twitter.com" class="fa fa-twitter"></a></li>
+                <li><a href="https://github.com/Nomarg/" class="fa fa-github"></a></li>
+            </ul>
+    </div>
```

Comment:
- [ ] Kindly wrap the "Headline" section in a `section` tag. It represents a generic standalone section of the document, so would be nice to use `section` tag. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.